src/annotator.cpp

```diff
@@ -574,7 +586,8 @@ ComponentPtr Annotator::component(const std::string &id, size_t index)
 {
     mPimpl->update();
     if (mPimpl->exists(id, index)) {
-        auto i = items(id).at(index);
+        auto privateIndex = realIndex(index);
+        auto i = items(id).at(privateIndex);
         return i->component();
```

Comment:
Why not simply (same elsewhere):
```c++
return items(id).at(realIndex(index))->component();
```
I know you do, but I really don't see the benefit of all those temporary variables.

Reply:
5d65515